modules/atomic-widgets/module.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Modules\AtomicWidgets\Widgets\Atomic_Heading;
+use Elementor\Plugin;
+use Elementor\Widgets_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+	const EXPERIMENT_NAME = 'atomic_widgets';
+	const EXPERIMENT_DEFAULT_STATE = Experiments_Manager::STATE_INACTIVE;
```

Comment:
Redundant, just use the const STATE_INACTIVE

Reply:
I use it in the tests to clean the environment.